### ISSUE TITLE
Questions about novel classes

### ISSUE THREAD

**Issue by chuanzeruge** [THU-MIG/yoloe#83]
Thank you for your excellent work！
As a beginner in OVD tasks， I have some questions.
1. When training by using visual prompts with a custom detection dataset (such as dotav1), how should I make the model  generate predictions of  nvoel classes during validation? Specifically, I manually divided dotav1 into 13 base classes and 2 novel classes, and removed the labels of the 2 novel classes in the training set. May I ask, according to YOLOE's training process, how can the probability of the novel class be output during the final verification?  By the way, I have changed the detection method of HBB to the OBB task according to the code.
2. After removing the labels of the two novel classes in the training set, I retained the labels of the two novel classes in the validation set. However, the final test results indicated that the model seemed not to have recognized these two novel classes because the results were both 0. May I ask if I should make modifications elsewhere?
